R/table-subscript.R

```diff
@@ -496,7 +497,7 @@ updateQStatisticsTestingInfo <- function(y, x.attributes, evaluated.args,
     ## Reorder q.test.info to be row-major by forming (correctly-ordered) indices
     ##  for output table and finding matches in original array indices
     perm <- rowMajorDimensionPermutation(new.dim.len, updated.qtypes)
-    if (new.dim.len > 1)
+    if (new.dim.len > 1 && all(lengths(dimnames(y)) > 0))
```

Comment:
Should the second part or this be `any(lengths(dimnames(y))>0`

Reply:
It needs to have fully defined names (none of the `dimnames` are `NULL`). `length(NULL)` is zero. So it requires all the lengths are bigger than zero.
e.g.
```
dimnames(y)
$Row
[1] "A" "B"
$Column
NULL
```
should not be enter the if statement.